resources/copy-to-clipboard.js

```diff
@@ -0,0 +1,144 @@
+/* PrismJS 1.29.0
```

Comment:
Just to confirm, this is basically a fork of the copy-to-clipboard plugin with some changes?

Reply:
If this is a copy, then it's nice to have a comment at the top saying so, with a link to the original, also mentioning the version of the original, and perhaps summarizing what was changed. So that in the future people could get a new version of the copied code and re-apply the changes.

Reply:
@malberts Yes, it's a fork. It copies only the visible lines in case of 'showLines' condition, in other cases all the lines same as before.

@JeroenDeDauw I have already mentioned the version number and functions that I changed at the start of the file, for the same purpose. As for the link, this cannot be directly downloaded, it comes coupled with the main Prismjs library, so one has to keep this unchecked (which it is, by default) so that it isn't included with the main library.

For reference here is the download page, core is not allowed to be unchecked:
![image](https://github.com/ProfessionalWiki/ExternalContent/assets/5208120/de1a4091-f246-4af2-b6c7-1c25601bbde6)

Here is the plugins section on the download page:
![image](https://github.com/ProfessionalWiki/ExternalContent/assets/5208120/07ebf457-faf4-428d-b64f-0097fed6042a)

Here is the prism.js file where in the second line it is mentioned which elements were selected on the download page, at the very end are the plugins:
![image](https://github.com/ProfessionalWiki/ExternalContent/assets/5208120/8264c261-2e23-46b1-8039-60adc7044db7)